### PR TITLE
feat(plugins): add menu-bar extension point

### PR DESCRIPTION
**Pull request by Aracturat**
Add new extension point to extend menu bar (other way is extraItems property).

### CODE REVIEW

File: lib/static/components/controls/menu-bar.js [hunk 1]
```diff
@@ -20,15 +22,20 @@ class MenuBar extends Component {
     render() {
         const {extraItems} = this.props;
 
-        if (isEmpty(extraItems)) {
+        const hasMenuBarPlugins = plugins.getLoadedConfigs()
+            .some(e => e.point === 'menu-bar');
```
Comment (DudaGod) [inline lib/static/components/controls/menu-bar.js:26]: let's define available extension points inside separate file here - https://github.com/gemini-testing/html-reporter/tree/master/lib/constants. And then use them here and where specify `result` extension point.

**Review by DudaGod — COMMENTED** [review]
Let's add tests for menu-bar component and I will merge it.

File: lib/static/components/controls/menu-bar.js (hunk 1 above)
Comment (Aracturat) [reply lib/static/components/controls/menu-bar.js]: Done

**Comment by Aracturat** [comment]
@DudaGod Added tests, please review and merge

**Comment by Aracturat** [comment]
@DudaGod Added tests, please review and merge

**Pull request merged**